gimme

```diff
@@ -286,6 +287,17 @@ _wipe_version() {
 	fi
 }
 
+_list_versions() {
+	if [ ! -d "${GIMME_VERSION_PREFIX}" ] ; then
+		return 0
+	fi
+
+	for d in "${GIMME_VERSION_PREFIX}/go"*".${GIMME_OS}."* ; do
+		local lessgo="${d##*/go}"
+		echo "${lessgo%%.${GIMME_OS}.*}"
+	done
```

Comment:
(since the output of `ls` is almost always user-centric, and can be heavily environment-dependent)